services/common/src/constants/API.ts

```diff
@@ -188,8 +188,6 @@ export const INFORMATION_REQUIREMENTS_TABLES = (projectGuid) =>
   `/projects/${projectGuid}/information-requirements-table`;
 export const INFORMATION_REQUIREMENTS_TABLE = (projectGuid, irtGuid) =>
   `/projects/${projectGuid}/information-requirements-table/${irtGuid}`;
```

Comment:
Long story short:
- separate endpoint for status has been nuked.
- status now combined with document endpoint (and also action creator, etc)
- caveat here is that additional parameters have to be added in form data (pretty obvious when you look at the action creator)